### PR TITLE
Fix/club model

### PR DESCRIPTION
**Pull request by mikefgalvin**
# Pull Request Template

## Description: 
- What work was done?
Adjusted the club model. Changed id to clubid. Changed location column to a club director column. 

- Why was this work done?
Update the model based on our new schema

- What feature / user story is it for?
User being able to pull club information 

- Any relevant links or images / screenshots

- User story link (Trello card)
https://trello.com/c/N7PvlpNR
## Type of change

- [ ] New feature
- [ ] Bug fix
- [ ] Design change
- [X] Modified existing code

## Minimum reviewers
- [X] 2 reviewers

## Testing
- [ ] Testing completed

### CODE REVIEW

**Review by justin-mavity — COMMENTED** [review]
Looks good thank you sir

File: src/main/java/com/lambdaschool/oktafoundation/models/Club.java [hunk 1]
```diff
@@ -5,40 +5,45 @@
...
     @GeneratedValue(strategy = GenerationType.AUTO)
-    private long id;
+    private long clubid;
 
     @NotNull
     @Column(unique = true)
     private String clubname;
 
-    private String location;
+    private int clubdirector;
```
Comment (ginabethrussell) [inline src/main/java/com/lambdaschool/oktafoundation/models/Club.java:24]: Change clubdirector variable to a string

File: src/main/java/com/lambdaschool/oktafoundation/models/Club.java [hunk 2]
```diff
@@ -5,40 +5,45 @@
...
+    @JsonIgnoreProperties(value = "club",
+            allowSetters = true)
+    private Set<ClubPrograms> roles = new HashSet<>();
 
     public Club()
     {
     }
 
-    public Club(@NotNull String clubname, String location)
+    public Club(@NotNull String clubname, int clubdirector)
```
Comment (ginabethrussell) [inline src/main/java/com/lambdaschool/oktafoundation/models/Club.java:38]: Change constructor for clubdirector variable string

File: src/main/java/com/lambdaschool/oktafoundation/models/Club.java [hunk 3]
```diff
@@ -48,12 +53,7 @@ public void setClubname(String clubname) {
...
     }
 
-    public String getLocation() {
-        return location;
-    }
-
-    public void setLocation(String location) {
-        this.location = location;
-    }
+    public int getClubdirector() { return clubdirector; }
```
Comment (ginabethrussell) [inline src/main/java/com/lambdaschool/oktafoundation/models/Club.java:56]: update getter and setters for clubdirector string

**Review by ginabethrussell — APPROVED** [review]
Great update to models. Just need to change clubdirector variable type. Will follow up with how to set this to a user.

**Pull request merged**